### ISSUE TITLE
Anonymous functions cause 'Unable to Open' error in VSCode

### ISSUE THREAD

**Issue by jonathan-spruce** [microclimate-dev2ops/microclimate-ls-node-prof#6]
In VSCode, when clicking on the details section of a problem where the parent caller was an anonymous function, it shows an error:
<img width="451" alt="screenshot 2019-02-11 at 18 05 02" src="https://user-images.githubusercontent.com/40430178/52583619-f0333200-2e27-11e9-9740-daa66b97d65d.png">

We probably just need to disable trying to link at all when the parent caller is an anonymous function.

**Comment by tetchel** [comment]
They still appear under the correct file, so why shouldn't they have the same URI as the other markers on that file? 
![image](https://user-images.githubusercontent.com/8888438/52584204-77b87b00-2dff-11e9-9a99-864d1871ae8f.png)

**Comment by jonathan-spruce** [comment]
You're right, and it also shows the line number it thinks it is. I'll do some more investigation because it looks like this should be working.

**Comment by tetchel** [comment]
The `(1, 1)` is default, it'll show even if you pass undefined or a line that doesn't exist. 

Unrelated but I think it's dumb that a position is always shown because not all warnings map to a position (such as my "missing files" errors in the Microclimate extension)
